src/pages/App.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import NavBar from '../components/NavBar';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 
 
 const App = React.createClass({
   render() {
     return (
-      <div>
-        <NavBar params={this.props.params}/>
-        <h1>Kodeklubben</h1>
-        {this.props.children}
-      </div>
+      <MuiThemeProvider muiTheme={getMuiTheme()}>
```

Comment:
Når det gjelder spørsmålet om MaterialUI kun skal være i de komponentene som trenger det, så vet jeg ikke nok om MUI ennå. Kom gjerne med argumenter for/imot.